lib/gssapi/mech/gss_mech_switch.c

```diff
@@ -462,6 +460,9 @@ _gss_load_mech(void)
 	}
 	fclose(fp);
 #endif
+
+out:
+	add_builtin(__gss_sanon_initialize());
```

Comment:
Nice way to make it last.  I'm thinking though that eventually it'd be nice to be able to have the gss config specify order of all mechanisms, even builtin ones, and then we'll need to do something about adding this builtin earlier, maybe.

Reply:
This is also arguably less necessary now we have mechanism order sorting.